src/compiler/semantics.c

```diff
@@ -1545,7 +1545,7 @@ static int process_enum(fb_parser_t *P, fb_compound_type_t *ct)
     fb_symbol_t *sym, *old, *type_sym;
     fb_member_t *member;
     fb_metadata_t *knowns[KNOWN_ATTR_COUNT];
-    fb_value_t index = { 0 };
+    fb_value_t index = { .len = 0 };
```

Comment:
ok. i pushed a more traditional version.

Reply:
Thanks. The downside to this is if the struct changes, zero init must be changed everywhere. I sometimes use memset for complex structs to avoid this. But I think this should be ok here.